src/protocol/scanner_cyrf6936.c

```diff
@@ -0,0 +1,162 @@
+/*
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Deviation is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Deviation.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "common.h"
+#include "interface.h"
+#include "pages/128x64x1/pages.h"
+
+#ifdef PROTO_HAS_CYRF6936
+
+#define MIN_RADIOCHANNEL    0x00
+#define MAX_RADIOCHANNEL    0x62
+#define CHANNEL_LOCK_TIME   300  // slow channel requires 270 usec for synthesizer to settle
+#define INTERNAL_AVERAGE    3
+#define AVERAGE_INTVL       50
+
+static struct scanner_page * const sp = &pagemem.u.scanner_page;
```

Comment:
we should not reference pagemem from a protocol. please find a different way to do so.

Reply:
I leaft struct in pagemem since it already was there. Otherwise we could move it into transmitter struct (or other suggestions?). But this would likely increase memory usage.